src/components/NewWorkspaceModal.js

```diff
@@ -221,16 +247,16 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
           isClearable: false,
           placeholder: 'Select a billing project',
           value: namespace,
+          ariaLiveMessages: ({ onFocus: onFocusAria, onChange: onChangeAria }),
           onChange: ({ value }) => setNamespace(value),
           styles: { option: provided => ({ ...provided, padding: 10 }) },
           options: _.map(({ projectName, invalidBillingAccount, cloudPlatform }) => ({
+            'aria-label': `${getCloudContextTitle(cloudPlatform)} ${projectName}${ariaInvalidBillingAccountMsg(invalidBillingAccount)}`,
```

Comment:
Unfortunately setting the `aria-label` is not sufficient to fix the issue because the default `ariaLiveMessages` does not use it (it always uses the label).